memory/allocator.go

```diff
@@ -293,28 +303,14 @@ func (a *GcAllocator) Free(b []byte) {
 
 }
 
-func (a *GcAllocator) Account(size int) error { return a.ResourceAllocator.Account(size) }
-
-func (a *GcAllocator) MaxAllocated() int64 {
-	return a.ResourceAllocator.MaxAllocated()
-}
+func (a *GcAllocator) Account(size int) error { return a.mem.Account(size) }
 
-func (a *GcAllocator) TotalAllocated() int64 {
-	return a.ResourceAllocator.TotalAllocated()
+func (a *GcAllocator) Allocated() int64 {
```

Comment:
These functions should be removed I think.

Reply:
They are needed for testing that we actually free things.